docs/conceptual/Getting_Started_FusionAPI.md

```diff
@@ -206,4 +206,6 @@ The following graph depicts the speedup gained for a fused Convolution+Bias+Acti
 
 Speedup obtained by fusing Batchnorm (spatial mode) with Activation are presented in the graph below:
 
-![Batchnorm activation fusion](data/na.png)
```

Comment:
you're in the conceptual folder, so if you want to reference something in the data folder, you need to use ".." in the path to 'go up' a folder. e.g. "[alt text](../data/conceptual/na.png)". Also note that you'll have to create the 'conceptual' folder within the data folder, as the data folder should have the same structure as the main docs folder. e.g. images that are referenced in the files in the tutorials folder should be present in a data/tutorials folder

Reply:
That's good to know. Do we know if this has been done before? Could you please share an example folder/template? Thanks so much.